bento2seldon4recsys/bento.py

```diff
@@ -135,10 +135,10 @@ def _send_feedback(
                 ]
                 logger.debug("relevance_scores: %s", relevance_scores)
 
-                ks = [request.jsonData.top_k]
-                if 10 < request.jsonData.top_k:
+                ks = [min(len(relevance_scores), request.jsonData.top_k)]
```

Comment:
We can initialize the `ks` list empty and then test if the list is longer than `request.jsonData.top_k`. If it is, we append to the list, if not, we don't report any metrics `@K`.